setup.py

```diff
@@ -0,0 +1,58 @@
+# installation: pip install dbsnap_verify
+from setuptools import (
+  setup,
+  find_packages,
+)
+
+setup( 
+    name = 'dbsnap_verify',
+    version = '0.0.2',
+    description = 'Tool for verifying AWS RDS snapshots.',
+    keywords = 'aws rds snapshot tool infrastructure',
+    long_description = open('README.rst').read(),
+
+    author = 'Russell Ballestrini',
+    author_email = 'russell@ballestrini.net',
```

Comment:
Since this is in remind101, I'd use your remind email here.

Reply:
fixed